ganga/GangaDirac/Lib/Backends/DiracUtils.py

```diff
@@ -204,3 +204,16 @@ def getAccessURLs(lfns, defaultSE = '', protocol = '', credential_requirements=N
     return myURLs
 
 exportToGPI('getAccessURLs', getAccessURLs, 'Functions')
+
+def getReplicas(lfns, credential_requirements=None):
+    """
+    Return a dict of files and their replicas.
+    lfns can be a string or a list
+    """
+    reps = execute('getReplicas(%s)' % str(lfns), cred_req=credential_requirements)
```

Comment:
This global function expects that `str(lfns)` will return a list of strings. Should it also be able to accept a dataset object?

Reply:
Good idea, sorted.